app/models/service_binding.rb

```diff
@@ -1,147 +1,57 @@
 require 'conjur_client'
 
-class ServiceBinding
-  include ConjurApiModel
-
+module ServiceBinding
   class RoleAlreadyCreated < RuntimeError
   end
 
   class HostNotFound < RuntimeError
   end
 
-  class << self
-    def create(instance_id, binding_id, org_guid, space_guid)
-      ServiceBinding.new(instance_id, binding_id, org_guid, space_guid).create
-    end
-
-    def delete(instance_id, binding_id, org_guid, space_guid)
-      ServiceBinding.new(instance_id, binding_id, org_guid, space_guid).delete
-    end
-  end
-
-  def initialize(instance_id, binding_id, org_guid, space_guid)
-    @instance_id = instance_id
-    @binding_id = binding_id
-    @org_guid = org_guid
-    @space_guid = space_guid
+  class NonExistentServiceBindingClass < RuntimeError
   end
 
-  def create
-    host = conjur_api.role(role_name)
-
-    raise RoleAlreadyCreated.new("Host identity already exists.") if host.exists?
-
-    begin
-      api_key = (ConjurClient.version == 4 ? create_host_v4 : create_host_v5)
-    rescue RestClient::NotFound => e
-      raise ConjurClient::ConjurAuthenticationError.new "Conjur configuration invalid: #{e.message}"
-    end
-
-    return {
-      account: ConjurClient.account,
-      appliance_url: ConjurClient.application_conjur_url,
-      authn_login: "host/#{host_id}",
-      authn_api_key: api_key,
-      ssl_certificate: ConjurClient.ssl_cert || "",
-      version: ConjurClient.version
+  # Look up table for available service bindings by
+  # Conjur version and enabled features (e.g. Space identity)
```

Comment:
This comment seems to indicate that we might create additional indices in this look-up table to handle other features being enabled / disabled? Do we have any plans to do this in the near future? If not we might want to just say explicitly that it's the space identity feature so it's a bit more concrete.

Reply:
Yeah, that makes sense. I guess I was accommodating for future feature flags that may come, but there's no reason to imply that if that don't currently exist :)